01-shapefile-attributes.Rmd

```diff
@@ -36,33 +36,35 @@ values.
 ###Goals / Objectives
 After completing this activity, you will:
 
- * Be able to query shapefile attributes
- * Be able to subset shapefiles based on specific attributes
+ * Be able to query shapefile attributes.
+ * Be able to subset shapefiles using specific attribute values..
+ * Know how to plot a shapefile, colored by unique attribute values.
  
-###What you'll need
-
-You will need the most current version of R or R studio loaded on your computer 
-to complete this lesson.
-
-###R Libraries to Install:
+###Things You'll Need To Complete This Lesson
+You will need the most current version of `R` and, preferably, RStudio loaded on
+your computer to complete this lesson.
 
+####R Libraries to Install:
 
-* **raster:** `install.packages("raster")`
-* **sp:** `install.packages("sp")`
 * **rgdal:** `install.packages("rgdal")`
-
+* **sp:** `install.packages("sp")`
```

Comment:
it's more justification to include it if they stumble on the page. A typical workflow for spatial data is checking in on things like extent and CRS. because those methods behave (or misbehave / don't work) differently depending on what libraries you have installed, it is a good idea for all of the vector lessons to include all libraries - and it's only 3. I am not sure what you are asking per the other lessons but for these, there is solid justification. If i were teaching this, id probably just suggest that people always load RGDAL and raster when working with spatial data. It is easier and more consistent.

Now level plot / raster vis for example is a very specific library / function that is only useful for time series. Thus, i woulnd't have them always load those. Raster - always load, rgdal, always load = good practice for spatial data IMO.